docs: polish sandbox tutorial (#4286)
jina-ai/jina: docs/how-to/sandbox.md

# How to use Jina Sandbox Executors

> In most cases, Jina Executor is just a docker image that contains some logic to process data. Therefore, you can run it locally if you have docker installed. But the docker image could be up to 1 or several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.

Reviewer: ```suggestion
It is often the case, that a Jina Executor is just a docker image that contains some logic to process documents. Therefore, you can run it locally if you have docker installed. But the docker image could be up to several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.
```

# How to use Jina Sandbox Executors

> It is often the case, that a Jina Executor is just a docker image that contains some logic to process documents. Therefore, you can run it locally if you have docker installed. But the docker image could be up to several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.

Reviewer: ```suggestion
It is often the case, that a Jina Executor is just a Docker image that contains some logic to process Documents. Therefore, you can run it locally if you have Docker installed. But the Docker image could be up to several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.
```

> Jina Sandbox provides a way to make the downloading and running happen in a cloud environment. It will get a pair of host and port, which you can connect with. Jina will automatically take care of this connection.

Reviewer: ```suggestion
Jina Sandbox provides a way to make the downloading and running happen in a cloud environment. It will give back a pair of host and port, which you can connect with. Jina will automatically take care of this connection.
```

There are some caveats when using Sandbox Executors with respect to other Executors.

### 1. Can't be controlled by the Flow

Since the lifetime of these Executors is not handled by the Flow and is handled by the Hub infrastructure, there is no way
to override its default configurations, therefore `uses_with`, `uses_metas`, etc ... will not apply.

You can consider a Sandbox Executor as an external Executor where you have no control over its initialization or configuration.

### 2. Don't support GPU yet

> Computation using GPU is most likely N times faster than using CPU in Machine Learning area. But unfortunately, GPU virtualization technology is not mature enough so we don't support using GPU to calculate in Sandbox container yet.

Reviewer: ```suggestion
Computation using GPU is most likely several times faster than using CPU in many usecases. But unfortunately,  we don't support using GPU for computation in Sandbox container yet.
```
